app/bundles/PageBundle/Form/Type/ConfigTrackingPageType.php

```diff
@@ -112,6 +112,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'data'  => (bool) $options['data']['google_analytics_landingpage_enabled'],
             ]
         );
+
+        $builder->add(
+            'google_analytics_anonymize_ip',
+            'yesno_button_group',
+            [
+                'label' => 'mautic.page.config.form.tracking.anonymize.ip.enabled',
+                'data'  => (bool) $options['data']['google_analytics_anonymize_ip'],
```

Comment:
I'm getting `PHP Notice - Undefined index: google_analytics_anonymize_ip` on this line when I apply this PR, clear the cache and access the Mautic configuration.

Reply:
Okay, issue depends on your fix https://github.com/mautic/mautic/pull/5663
I added param to ConfigSubsriber. Now should works

Reply:
That fix is merged already so it should be included in your PR too. The problem is that the $options['data'] doesn't contain the new property yet.

Reply:
But this line fixed it, didn't it? https://github.com/mautic/mautic/pull/5525/commits/d763b0a14ca4f2129dd7b9313455010761111147#diff-2ede0d259417f4d170e6c0f8b2cbf442R71

Reply:
I don't think so. Try to remove that value from your app/config/local.php, clear the cache and try to load the configuration.